### PR TITLE
Adding new Riedel devices

### PR DESCRIPTION
**Pull request by moldennrk**

### CODE REVIEW

File: elevation-images/Riedel/riedel-esp-1216hl-front.png
Comment (sjurtf) [inline elevation-images/Riedel/riedel-esp-1216hl-front.png:1]: elevantion-images are used to show the device inside a rack view.
The image should only contain the device and not empty space around it.

Could you please modify the images and remove the empty space around the device? Unfortently you might have to crop away the microphone in some of these images as well.

For the images to be included in the import, you must add the following to the corresponding device type yaml file.

```yml
front_image: true
rear_image: true
```

**Review by sjurtf — CHANGES_REQUESTED** [review]

File: device-types/Riedel/artist-1024.yaml [hunk 1]
```diff
@@ -7,30 +7,37 @@ is_full_depth: false
 airflow: front-to-rear
 weight: 6.3
 weight_unit: kg
-front_image: true
```
[the remark this answers is not in the document]
Comment (moldennrk) [reply device-types/Riedel/artist-1024.yaml:10]: These files were based on yaml export from netbox. The image properties are not included in this export.

File: device-types/Riedel/artist-32.yaml [hunk 2]
```diff
@@ -5,23 +5,27 @@ slug: riedel-artist-32
 u_height: 2.0
 is_full_depth: false
 airflow: left-to-right
-front_image: true
-rear_image: true
```
[the remark this answers is not in the document]
Comment (moldennrk) [reply device-types/Riedel/artist-32.yaml]: These files were based on yaml export from netbox. The image properties are not included in this export. Fixed in new commit.

File: device-types/Riedel/artist-128.yaml [hunk 3]
```diff
@@ -5,47 +5,51 @@ slug: riedel-artist-128
 u_height: 6.0
 is_full_depth: false
 airflow: front-to-rear
-front_image: true
-rear_image: true
```
[the remark this answers is not in the document]
Comment (moldennrk) [reply device-types/Riedel/artist-128.yaml]: These files were based on yaml export from netbox. The image properties are not included in this export. Fixed in new commit.

File: device-types/Riedel/artist-64.yaml [hunk 4]
```diff
@@ -5,31 +5,35 @@ slug: riedel-artist-64
 u_height: 3.0
 is_full_depth: false
 airflow: front-to-rear
-front_image: true
-rear_image: true
```
[the remark this answers is not in the document]
Comment (moldennrk) [reply device-types/Riedel/artist-64.yaml]: These files were based on yaml export from netbox. The image properties are not included in this export. Fixed in new commit.

File: device-types/Riedel/bl-ant-1010-19g-eu-g2.yaml [hunk 5]
```diff
@@ -0,0 +1,39 @@
+---
+manufacturer: Riedel
+model: BL-ANT-1010-19G-EU G2
+slug: riedel-bl-ant-1010-19g-eu-g2
+description: Riedel Bolero antenne
```
[the remark this answers is not in the document]
Comment (moldennrk) [reply device-types/Riedel/bl-ant-1010-19g-eu-g2.yaml]: Description translated.

File: device-types/Riedel/rsp-1216hl.yaml [hunk 6]
```diff
@@ -0,0 +1,99 @@
+---
+manufacturer: Riedel
+model: RSP-1216HL
+slug: riedel-rsp-1216hl
+description: Kommunikasjonspanel 16 knapper
```
[the remark this answers is not in the document]
Comment (moldennrk) [reply device-types/Riedel/rsp-1216hl.yaml]: Description translated.

File: device-types/Riedel/rsp-1232hl.yaml [hunk 7]
```diff
@@ -0,0 +1,99 @@
+---
+manufacturer: Riedel
+model: RSP-1232HL
+slug: riedel-rsp-1232hl
+description: Kommunikasjon panel med 32 knapper og sterio
```
[the remark this answers is not in the document]
Comment (moldennrk) [reply device-types/Riedel/rsp-1232hl.yaml]: Description translated.

File: device-types/Riedel/bl-bpk-1006-19-eu.yaml [hunk 8]
```diff
@@ -0,0 +1,15 @@
+---
+manufacturer: Riedel
+model: BL-BPK-1006-19-EU
+slug: riedel-bl-bpk-1006-19-eu
+description: Riedel Bolero 6-knapper handsett
```
[the remark this answers is not in the document]
Comment (moldennrk) [reply device-types/Riedel/bl-bpk-1006-19-eu.yaml]: Description translated.

File: elevation-images/Riedel/riedel-esp-1216hl-front.png
Comment (moldennrk) [reply elevation-images/Riedel/riedel-esp-1216hl-front.png:1]: Fixed.

**Review by sjurtf — APPROVED** [review]

**Pull request merged**